extensions/strategies/stddev/strategy.js

```diff
@@ -0,0 +1,53 @@
+//" sudo ./zenbot.sh sim --strategy=stddev --trendtrades_1=100 --trendtrades_2=300 --min_periods=1005 --period=100ms";
+var z = require('zero-fill')
+var stats = require('stats-lite')
+var n = require('numbro')
+var tl0mp = []
+module.exports = function container (get, set, clear) {
+  return {
+    name: 'stddev',
+    description: 'Trade when % change from last two 1m periods is higher than average.',
+
+    getOptions: function () {
+      this.option('period', 'period length', String, '100ms')
+      this.option('trendtrades_1', "Trades for trend 1", Number, 100)
+      this.option('trendtrades_2', "Trades for trend 1", Number, 300)
```

Comment:
Should this be: `this.option('trendtrades_2', "Trades for trend 2", Number, 300)`?